### PR TITLE
Heading component

### PR DESCRIPTION
**Pull request by rodriguez-gaston-simplenight**
https://simplenight.atlassian.net/browse/SAF-1870

[SAF-1870]: https://simplenight.atlassian.net/browse/SAF-1870?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/Heading/Heading.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+
+import Heading from './Heading';
+
+const title = 'SimplenightUI/';
```
Comment (NakaSimpleNight) [inline src/components/Heading/Heading.stories.tsx:6]: We could extract this const to a constant folder just to avoid declaring it

File: src/components/Heading/TypographyTypes.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
+type ColorName = 'primary' | 'dark' | 'green';
```
Comment (NakaSimpleNight) [inline src/components/Heading/TypographyTypes.ts:1]: It would be nice to add both error (red) and warning (yellow). Of course, it depends on how we are going to implement it in the config file.

File: src/components/Heading/Heading.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  tag: 'h1',
+  children: 'Heading Test',
+};
+
+export const HeadingPrimary = Template.bind({});
+HeadingPrimary.args = {
+  tag: 'h1',
+  children: 'Heading Test',
+  textColor: 'text-primary-1000',
+};
```
Comment (NakaSimpleNight) [inline src/components/Heading/Heading.stories.tsx:28]: Could we add more cases just to have better documentation?

File: src/components/Heading/Heading.test.tsx [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import Heading from './Heading';
+
+describe('Heading', () => {
+  test('Renders the heading component', () => {
```
Comment (NakaSimpleNight) [inline src/components/Heading/Heading.test.tsx:6]: A good test to add could be to test the correct usage of properties. Or within the property text-color, the dev can not pass another tailwind class.

File: src/components/Paragraph/Paragraph.tsx [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+}: ParagraphProps) => {
+  const paragraphClass = paragraphClasses[size];
+
+  return (
+    <p
+      className={`${paragraphClass} ${textColor} ${className} ${
+        fontWeight === 'semibold' && 'font-semibold'
+      } ${fontWeight === 'normal' && 'font-normal'} ${
+        fontWeight === 'medium' && 'font-medium'
+      } font-lato m-0`}
```
Comment (NakaSimpleNight) [inline src/components/Paragraph/Paragraph.tsx:19]: Could we extract the logic outside the <p> element?

File: tailwind.config.js [hunk 6]
```diff
@@ -1,114 +1,131 @@
...
-            boxShadow: {
-                container: '0 0px 12px rgba(0, 0, 0, 0.08)',
-            },
+  content: ['./src/**/*.{js,jsx,ts,tsx}', './src/stories/*'],
+  safelist: [
+    { pattern: /primary/, variants: ['hover'] },
+    { pattern: /dark/, variants: ['hover'] },
+    { pattern: /border/ },
+    'bg-transparent',
+  ],
```
Comment (NakaSimpleNight) [inline tailwind.config.js:10]: Why is this added? This could potentially increase build time.

**Review by NakaSimpleNight — CHANGES_REQUESTED** [review]

File: tailwind.config.js (hunk 6 above)
Comment (rodriguez-gaston-simplenight) [reply tailwind.config.js:10]: This was added two months ago by Enrique. The only change in these lines was fixed the indentation with eslint
Screen Shot 2023-01-03 at 12 31 15

**Review by NakaSimpleNight — APPROVED** [review]
